client/src/pages/CreateWorkout/CreateWorkout.jsx

```diff
@@ -3,63 +3,73 @@ import VideoCard from "@client/src/components/VideoCard/VideoCard";
 import { useState } from "react";
 import { useLocation, useNavigate } from "react-router-dom";
 import { useAuth0 } from "@auth0/auth0-react";
-import WorkoutContext from "@client/src/contexts/workout";
+import {
+  postWorkout,
+  updateWorkout,
+  deleteWorkout,
+  getWorkout,
+  getWorkouts,
+} from "@client/src/apis/WorkoutAPI";
 import ListExercises from "./components/ListExercises/ListExercises";
 
 const CreateWorkout = () => {
   const location = useLocation();
+  const navigate = useNavigate();
   const videoInfo = location.state;
   const { workoutId } = location.state;
-  console.log(workoutId);
-  const [newWorkout, setNewWorkout] = useState(false);
-
-  const {
-    workout,
-    exercises,
-    setWorkout,
-    getWorkout,
-    deleteWorkout,
-    postWorkout,
-    updateWorkout,
-  } = useContext(WorkoutContext);
-
   const { user, isAuthenticated } = useAuth0();
 
-  const [targetArea, setTargetArea] = useState(
-    workout?.target_area || "full-body"
-  );
-
-  console.log(targetArea);
+  const [workout, setWorkout] = useState({
+    user_id: "",
+    video_id: "",
+    target_area: "full-body",
+    exercises: [],
+  });
 
-  const navigate = useNavigate();
+  const handleChange = (e) => {
+    setWorkout(prevWorkout => ({ ...prevWorkout, target_area: e.target.value }));
+  };
 
   useEffect(() => {
-    if (user) {
-      getWorkout(workoutId, isAuthenticated);
-    }
+    const workout = async () => {
```

Comment:
would advise not naming this something else like `loadWorkout` to avoid a name conflict / any confusion with your `workout` state.